<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu_toolbar</class>
 <widget class="QWidget" name="menu_toolbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;
border-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QPushButton" name="btn_messages">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>180</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>messages</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_logoff">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>33</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>logoff</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_menu">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>180</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>menu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_archive">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>archive</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_alert">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>26</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>alert</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
